src/main/java/com/conveyal/datatools/manager/DataManager.java

```diff
@@ -159,10 +167,26 @@ static void initializeApplication(String[] args) throws IOException {
         feedBucket = getConfigPropertyAsText("application.data.gtfs_s3_bucket");
         bucketFolder = FeedStore.s3Prefix;
 
+        // create application gtfs folder if it doesn't already exist
+        new File(getConfigPropertyAsText("application.data.gtfs")).mkdirs();
```

Comment:
When I'm using the dev configuration it lists the gtfs directory in `tmp` which I guess is getting deleted every so often.  Therefore, I added it in here so it will always be created if the folder doesn't exist.  The current code results in the application noting an error on startup, but not exiting, but then once a webservice is called all kinds of errors happen.  I think it'd be good default behavior to create this folder if it doesn't exist.

Reply:
What if someone is using a shared config file and the path is set to `/Users/landon/nested/dir/that/is/very/deeply/buried`. Do we really want to create those directories on the server? If the application isn't properly failing fast, let's just add that behavior. We can also just change the default gtfs directory if that behavior is annoying.

Reply:
Regarding creating supposedly random folders on the server, if it's what the user specified, why wouldn't datatools-server do that?  I don't even know what goes in that folder in the first place.  I actually like that the files are saved to a tmp directory so that they get wiped out locally so I don't have to periodically clean things up.  I'm also not a fan of the server failing because I haven't done even more manual setup of things.  It should just work.

Reply:
It is arbitrary how a server reacts when it's misconfigured. It can fail fast or attempt to recover by creating missing directories. The important thing is that we choose a behavior and stick to it. I'm increasingly a fan of failing fast and showing clear error messages. Every error recovery code path is another thing that needs to be maintained and can have unexpected consequences, e.g. if someone misspells the intended path and we silently create it, or attempt to create it and it fails with a seemingly irrelevant permissions error.